public/static/locales/de/common.json

```diff
@@ -14,7 +14,7 @@
   "companyName": "Klinik / Unternehmen",
   "components": {
     "approbationSection": {
-      "title": "Approbationen"
+      "title": "Approbatio\u00ADnen"
```

Comment:
what happened here? Seems unintentional. (same for the other translation changes)